9.2/run-postgresql.sh

```diff
@@ -1,12 +1,23 @@
 #!/bin/bash
 
+# Set HOME variable
+export HOME=/var/lib/pgsql
+
 # For SCL enablement
 source $HOME/.bashrc
 
 set -eu
 
 # Data dir
-export PGDATA=$HOME/data
+if [ -O $HOME/data ]
+then
+  export PGDATA=$HOME/data
+else
+  # If current user does not own data directory
+  # create a subdirectory that the user does own
+  mkdir $HOME/data/userdata
+  export PGDATA=$HOME/data/userdata
```

Comment:
How will the dir be created if the user does not own $HOME/data? Also, I think that in this case, we should fail.